features/calculation_weekly_frequency.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  As a enrolee into a Workplace Pension,
+  I would like to calculate pension contribution on my weekly salary.
+
+  Background:
+    Given I am on the YourDetailsPage
+
+    Scenario Outline:
+      Given I enter my age as "<age>"
```

Comment:
this would move a lot of the logic into the steps and keep the scenarios concise..
wdyt?